exercises/rotational-cipher/src/example/java/RotationalCipher.java

```diff
@@ -0,0 +1,29 @@
+public class RotationalCipher {
+    public String rotate(String data, int  shiftKey){
+        String cipherText = "";
+        for(int i = 0;i < data.length(); i++){
+            //for uppercase characters
+            if(data.charAt(i) >= 'A' && data.charAt(i) <= 'Z'){
+                char replacementCharacter = (char) (data.charAt(i) + shiftKey);
+                if(replacementCharacter > 'Z'){
+                    replacementCharacter = (char)(64 + (replacementCharacter % 90));
+                }
+                cipherText += replacementCharacter;
+            }
+            //for lowercase characters
+            else if(data.charAt(i) >= 'a' && data.charAt(i) <= 'z'){
```

Comment:
These `if` and `else` statements contain some duplication of code. Maybe you can pull the duplicated code out into a method? :)